h2o-core/src/main/java/water/fvec/Vec.java

```diff
@@ -606,6 +606,7 @@ public static Vec makeVec(long [] vals, String [] domain, Key<Vec> vecKey){
    *  and initialized to the given constant value.  */
   public Vec makeCon(final double d) { return makeCon(d, group(), _rowLayout, T_NUM); }
   public Vec makeCon(final double d, byte type) { return makeCon(d, group(), _rowLayout, type); }
+  public Vec makeCon(final long l, byte type) { return makeCon(l, null, group(), _rowLayout, type); }
```

Comment:
I don't think this works in the general case... if your goal is to create a constant NA columns with String please add a method without the `long` parameter, eg. `makeConStr(byte type)`

this creates a Vec made of `C0DChunks`

please also add a test that shows that rollups can be calculated over such column

Reply:
I think this will work. What you are suggesting is pretty much what 

public Vec makeCon(final double d, byte type) 
https://github.com/h2oai/h2o-3/pull/3277/files/3957805df8a43235c18228836d37a0b6111d6410#diff-8087560966ac7843cf17ceb4410c0be2L607

does, but inside is a check
if( (long)d==d ) return makeCon((long)d, null, group, rowLayout, type)

so it ends up creating C0LChunk chunks, so we should be fine.

I have changed the API to hide the 0L inside the call. Added a test as well.